### ISSUE TITLE
✨ Turn on Career planning widget on Applicant Dashboard

### ISSUE THREAD

**Issue by tristan-orourke** [GCTC-NTGC/gc-digital-talent#12332]
## ✨ Feature

This ticket is just about turning on the Career Planning widget once all related work is complete. 

The existence of this ticket is to make parallelization of other tickets easier.

## 📸 Screenshot

![image](https://github.com/user-attachments/assets/c0220d27-b279-42cd-84e9-b142900d98cf)

## ✅ Acceptance Criteria

- [ ] After this is merged & deployed, gov employees with verified work emails will see the new widget on their dashboard.

## 🛑 Blockers

```[tasklist]
### Blocked By
- [ ] #12284 
- [ ] #12288 
- [ ] #12285
```

**Comment by vd1992** [comment]
@petertgiles 

I don't think this issue is necessary or does anything now. The widget is active but hidden by an inaccessible route

**Comment by petertgiles** [comment]
Decision in standup: issue not needed, page can launch with the widget
